README.md

```diff
@@ -1,13 +1,41 @@
-# wg-ai-gateway
+# AI Gateway Working Group
 
-Proposals and discussions for the [AI Gateway Working Group](https://github.com/kubernetes/community/tree/master/wg-ai-gateway).
+Proposals and discussions for the [AI Gateway Working Group].
+
+[AI Gateway Working Group]:https://github.com/kubernetes/community/tree/master/wg-ai-gateway
+
+## About
+
+For more general information about the working group, please see:
+
+* Our [README] - This provides an overview of the community, contact and
+  communication information.
+* Our [Charter] - This describes what this WG intends to accomplish in
+  detail, our general goals and areas of interest.
+
+As a Kubernetes working group, we do not _directly_ own projects or code,
+our purpose is to make proposals. As such, this repository is meant for
+discussions and staging proposals which (generally) will be developed into
+proposals for specific Kubernetes Special Interest Groups (SIGs) and their
+sub-projects.
+
+[README]:https://github.com/kubernetes/community/blob/master/wg-ai-gateway/README.md
+[Charter]:https://github.com/kubernetes/community/blob/master/wg-ai-gateway/charter.md
+
+### Proposals
+
+Please see our [proposals directory](/proposals) to view the proposals we
+have so far. If you'd like to submit a new proposal, please join us in the
+community to get some feedback and identify potential supporters for the
+proposal in the community who can help move it forward with you.
 
 ## Community, discussion, contribution, and support
 
 Learn how to engage with the Kubernetes community on the [community page](http://kubernetes.io/community/).
 
-You can reach the maintainers of this project at:
+You can reach the maintainers of this working group at:
```

Comment:
WG has chairs, not maintainers, isn't it?
```suggestion
You can reach the chairs of this working group at:
```

Reply:
Well I think in this case, the template uses the maintainer language because it's literally referring to the maintainers of the _repository_, but given what our repository is and what we're trying to communicate here it is a little off. I'm going to try this instead, LMKWYT:

```suggestion
You can reach out to members of the community at:
```